opengever/maintenance/scripts/persist_default_value_for_paperform.py

```diff
@@ -0,0 +1,106 @@
+from logging import Formatter
+from logging import getLogger
+from logging import INFO
+from logging import StreamHandler
+from opengever.base.default_values import object_has_value_for_field
+from opengever.document.behaviors.metadata import IDocumentMetadata
+from opengever.document.behaviors.metadata import preserved_as_paper_default
+from opengever.document.document import IDocumentSchema
+from opengever.maintenance.debughelpers import setup_app
+from opengever.maintenance.debughelpers import setup_option_parser
+from opengever.maintenance.debughelpers import setup_plone
+from plone import api
+from time import time
+import gc
+import transaction
+
+
+def fire_gc():
+    start = time()
+    logger.info('Firing garbage collection')
+    api.portal.get()._p_jar.cacheGC()
+    gc.collect()
+    logger.info('Garbage collection done in %.3fs', time() - start)
+
+
+def persist_default_value_for_paperform():
+    catalog = api.portal.get_tool('portal_catalog')
+    query_filter = {
+        'object_provides': IDocumentSchema.__identifier__,
```

Comment:
Needs to be `IBaseDocument`, mails has the same field.

Reply:
Will amend.